kong/plugins/datadog/schema.lua

```diff
@@ -0,0 +1,8 @@
+return {
+  fields = {
+    host = {type = "string", default = "localhost"},
+    port = {type = "number", default = 8125},
+    metrics = {type = "array", enum = {"request_count", "latency", "request_size", "status_count"}, default = {"request_count", "latency", "request_size", "status_count"}}, 
+    timeout = {type = "number", default = 10000}
```

Comment:
No, user can decide to try to remove the value...

> On Dec 19, 2015, at 7:08 AM, shashi ranjan <notifications@github.com> wrote:
> 
> In kong/plugins/datadog/schema.lua:
> 
> > @@ -0,0 +1,8 @@
> > +return {
> > +  fields = {
> > +    host = {type = "string", default = "localhost"},
> > +    port = {type = "number", default = 8125},
> > +    metrics = {type = "array", enum = {"request_count", "latency", "request_size", "status_count"}, default = {"request_count", "latency", "request_size", "status_count"}}, 
> > +    timeout = {type = "number", default = 10000}
> if we passing default value then why we need to make required? it will always has the default value unless user provide another value
> 
> —
> Reply to this email directly or view it on GitHub.
>